src/Components/Applications/Form/index.js

```diff
@@ -0,0 +1,175 @@
+import { useEffect, useState } from 'react';
+import styles from './form.module.css';
+function Form() {
+  const [positionId, setPositionId] = useState('');
+  const [postulantId, setPostulantId] = useState('');
+  const [date, setDate] = useState('');
+  const [result, setResult] = useState('');
+  const [positions, setPositions] = useState([]);
+  const [postulants, setPostulants] = useState([]);
+  const [interviews, setInterviews] = useState([]);
+
+  const params = new URLSearchParams(window.location.search);
+  const applicationId = params.get('_id');
+  let fetchMethod = 'POST';
+  let disableButton = false;
```

Comment:
it is wrong, your component will not be updated when this var is changed. For this reason, you cannot currently edit an application in your PR.

Reply:
But the reason i can't update an application is that there is no route or controller to edit on the server. Thats why i add that var. Either way i will correct it to use it as a state